controller/index.js

```diff
@@ -437,13 +438,15 @@ var Controller = function (agol, BaseController) {
         if (err) {
           agol.log('error', 'Failed to get count of rows in the DB' + ' ' + err)
           // don't let db messages leak out
-          return res.status(500).json({error: 'Unknown failure'})
         }
         // if we have a passed in error or the info doc says error then this request is errored and we should send a 502 with status failed
         var errored = (error && error.message) || (info.generating && info.generating.error)
         var code = errored ? 502 : 202
         var status = errored ? 'Failed' : (info.status || 'Processing')
-
+        if (!info) {
+          info = {retrieved_at: Date.now()}
+          status = 'Failed'
+        }
```

Comment:
why are you defining status and then immediately redefining it?
also if `!info` evaluates to true then info.status may throw an uncaught error above